internal/entity/photo.go

```diff
@@ -911,7 +911,7 @@ func (m *Photo) SetCoordinates(lat, lng float32, altitude int, source string) {
 		return
 	}
 
-	if SrcPriority[source] < SrcPriority[m.PlaceSrc] && m.HasLatLng() {
```

Comment:
`m.HasLatLng()` is there so that missing information can be added from sources that don't have priority - or `PlaceSrc` is set incorrectly, e.g. because people messed with their database.

Reply:
I dont know about the other users, but if I personally notice that some of my photos are geo-tagged incorrectly, I will not fix the location directly in PhotoPrism, but rather fix the geotag in the photo itself. With the existing implementation, the updated photo location will be ignored, as the photo was previously geotagged. This is basically the only reason why I removed the `m.HasLatLng()` check.

Reply:
This is strange as we fixed and tested this, probably before our release in December. Did you change the location manually in the UI as well? That won't be overwritten by changes in the file metadata.

The lookup table clearly shows that metadata has a much higher priority than estimates:

https://github.com/photoprism/photoprism/blob/develop/internal/entity/src.go#L22

Reply:
Nevermind, I cant read 🤦  I tested it with the latest `develop` and works as expected...